packages/utils/theme/src/defaultTheme.ts

```diff
@@ -199,6 +199,7 @@ export const defaultTheme: Theme = {
   },
   spacing: {
     auto: 'auto',
+    current: '1em',
```

Comment:
Adding it to the `spacing` namespace of the tailwind's theme definition might be "too much".

Spacing is used as a basis for the following TW namespaces:

- borderSpacing
- flexBasis
- gap
- height
- inset
- margin
- maxHeight
- padding
- scrollMargin
- scrollPadding
- space
- textIndent
- translate
- width

Adding it to `spacing` means we make it available to all those namespaces, so Spark users would be able to use the following classes:

```
p-current, m-current, gap-current, etc
```

Is it an issue or not ? that is the question :D (All those values will appear in autocomplete of people using Spark)

Reply:
Maybe "base"?

Reply:
Or we could add a dedicated class just for our use case, something like `.icon-base-size`, `.icon-base` or whatever 😄
WDYT, @Powerplex  @andresz1 ?

```css
.icon-base-size { font-size: 1rem; }
```

Reply:
Oh, ok, so then we could do:
```css
.icon-base-size { width: 1em; height: 1em; }
```

Reply:
LGTM @acd02 I will merge this PR since we agreed to change it in another PR

Reply:
`em` as a unit refers to the font-size. So if you set it as a token inside `spacing`, we add it to all the namespace I listed. So it should be clear by the name that it refers to a unit relative to the font-size (em) or it can be confusing for the users.

"current" is maybe too generic, if we see `p-current`, it is not clear that padding will be 1em (unit related to the base font size of the parent)

Other naming suggestions: 
`p-current-font` OR `p-1em`

(That is if we decide to keep it inside of `spacing`)

Reply:
We are going to add a specific utility class to handle that `.icon-base-size` @Powerplex , so we won't be updating our theme